src/monster.cpp

```diff
@@ -774,7 +774,9 @@ void Monster::doAttacking(uint32_t interval)
 
 	for (const spellBlock_t& spellBlock : mType->info.attackSpells) {
 		bool inRange = false;
-
```

Comment:
This `if` should be separated by empty lines above and below like the rest of the code. Don't change the formatting.